src/dotnet-interactive-vscode/src/interactiveClient.ts

```diff
@@ -234,7 +225,15 @@ export class InteractiveClient {
                     listener(eventEnvelope);
                 }
             } else {
-                this.deferredCommandEventListener(eventEnvelope);
+                switch (eventEnvelope.eventType) {
+                    case DisplayedValueProducedType:
+                    case DisplayedValueProducedType:
```

Comment:
You have a duplicate here. Did you mean `DisplayedValueUpdatedType`?

Also is there a test for this?

Reply:
Test added.